classes/student.rb

```diff
@@ -1,4 +1,6 @@
 class Student < Person
+  attr_reader :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
```

Comment:
- [ ] It is a requirement that When setting the classroom for a student it also adds it to the classrooms' students:
e.g
``classroom.students.push(self) unless classroom.students.include?(self)``